pkg/models/runtime_env_label.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package models
+
+import (
+	"fmt"
+
+	"openpitrix.io/openpitrix/pkg/utils"
+)
+
+const RuntimeEnvLabelTableName = "runtime_env_label"
+
+func NewRuntimeEnvLabelId() string {
+	return utils.GetUuid("rel-")
+}
+
+type RuntimeEnvLabel struct {
+	RuntimeEnvLabelId string
```

Comment:
The code is not formatted. Please `git commit` after `make fmt-all`

Reply:
done